src/main/java/net/comments/ft/Test4.java

```diff
@@ -0,0 +1,29 @@
+package net.comments.ft;
+
+import net.comments.objects.*;
+import org.hamcrest.MatcherAssert;
+import org.testng.annotations.Test;
+
+public class Test4 extends BaseDriver {
+    private final CommentPage page;
+    private final CommentActions action;
+    private final BCommentTable table;
+
+    public Test4() {
+        super();
+        this.page = new BCommentPage(this.driver);
+        this.action = new BCommentActions(this.driver);
+        this.table = new BCommentTable(this.driver);
+    }
+
+    @Test
+    public void test() {
+        int commentID = 1;
+        page.open();
+        action.delete(commentID);
+        table.fillTable();
+        for (Comment comment : table.getTableComments()) {
```

Comment:
**table**.get**Table**Comments() - do you have some other then table comments? Tautology.

Reply:
@extsoft getTableComments() - this method return table with comments. Maybe I need to rename this method?

Reply:
@oteplova yes, please rename.